pyproject.toml

```diff
@@ -0,0 +1,92 @@
+[build-system]
+requires = [
+    "hatchling>=1.13.0",
+    "hatch-vcs>=0.3.0",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "pyshaper"
```

Comment:
The name `pyshaper` has been chosen here, but this was just thrown out as an example name given that https://pypi.org/project/shaper already exists. Alternative names are welcome and easy to swap to!

Reply:
In hindsight, maybe we should have checked this before naming our framework! Checked with collaborators, `pyshaper` is a good name.

Reply:
Sounds good. The `pyshaper` name is currently available on TestPyPI (https://test.pypi.org/project/pyshaper/) and PyPI (https://pypi.org/project/pyshaper/).